### PR TITLE
mstarch: cleaning up issues detected by LGTM

### PR DESCRIPTION
**Pull request by LeStarch**
Cleans up a number of changes required by LGTM.  Also removes wxgui GDS.  This can be found in release 1.5.3.

### CODE REVIEW

File: Autocoders/Python/src/fprime_ac/generators/formatters.py [hunk 1]
```diff
@@ -1413,7 +1413,7 @@ def argFilter(self, module, args, name="foo"):
                 new_args.append((id2, "U16", comment, "", "", [], []))
 
             elif type == "SCLK":
-                type = module[0].upper() + module[1:] + "ArgSclk"
+                _ = module[0].upper() + module[1:] + "ArgSclk"
```
Comment (Joshua-Anderson) [inline Autocoders/Python/src/fprime_ac/generators/formatters.py:1416]: Can't we just delete this line?

File: Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ def DictStartVisit(self, obj):
         output_dir = os.environ["DICT_DIR"] + "/channels"
         if not (os.path.isdir(output_dir)):
             os.makedirs(output_dir)
-        pyfile = output_dir + "/" + obj.get_name() + ".py"
+        _ = output_dir + "/" + obj.get_name() + ".py"
```
Comment (Joshua-Anderson) [inline Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py:97]: same here

**Review by Joshua-Anderson — CHANGES_REQUESTED** [review]
There's quite a few places where you're assigning to _, which is a no-op (aside from the mutation from the assignment itself).

Nearly all of these lines can just totally be removed, and a new of them could probably be refactored to avoid the assignment.

File: Autocoders/Python/src/fprime_ac/generators/formatters.py (hunk 1 above)
Comment (LeStarch) [reply Autocoders/Python/src/fprime_ac/generators/formatters.py:1416]: I made a lot of switches from variable -> _ but left the line in.  Given the overall quality of the code base, I cannot be sure that there aren't side-effects to function calls and what not.

Thus clean ups more substantial will require a full refactor.

File: Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py (hunk 2 above)
Comment (LeStarch) [reply Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py:97]: I made a lot of switches from variable -> _ but left the line in.  Given the overall quality of the code base, I cannot be sure that there aren't side-effects to function calls and what not.

Thus clean ups more substantial will require a full refactor.

Comment (Joshua-Anderson) [reply Autocoders/Python/src/fprime_ac/generators/visitors/ChannelVisitor.py:97]: Yeah, there may be some unexpected affects, but if that's the case can't you just directly call the functions you're concerned have side-effects?

Also, just scrolling through the changes, most of them seem this one, where removing the entire line seems pretty safe (unless get_name is doing something insane).

I'd argue this cleanup is pointless without actually removing the offending lines (other than appeasing the linter), but if it doesn't make sense to invest the effort in removing these lines now, I guess at least clearly marks the lines for cleanup later.

**Comment by LeStarch** [comment]
Agreed to fix the obvious assignment only statements.

**Comment by LeStarch** [comment]
@Joshua-Anderson  should be ready for re-review

**Review by Joshua-Anderson — APPROVED** [review]

**Pull request merged**